modules/installation-osp-verifying-cluster-status.adoc

```diff
@@ -7,6 +7,8 @@
 [id="installation-osp-verifying-cluster-status_{context}"]
 = Verifying cluster status
 
+// Should this step be part of the installation paths? Let's verify.
```

Comment:
@tomassedovic @iamemilio Are the steps in this module important important enough to include in the typical deployment flows?

Reply:
I think after the installer exits, it is helpful to check. It is possible that something like a worker node failed to stand up, but the installer just plowed through it.